src/test/java/org/mitre/synthea/export/FHIRDSTU2ExporterTest.java

```diff
@@ -45,26 +59,24 @@ public void testFHIRDSTU2Export() throws Exception {
     validator.setValidateAgainstStandardSchematron(true);
 
     List<String> validationErrors = new ArrayList<String>();
-    
+
     int numberOfPeople = 10;
     Generator generator = new Generator(numberOfPeople);
     for (int i = 0; i < numberOfPeople; i++) {
       int x = validationErrors.size();
-      TestHelper.exportOff();
```

Comment:
You need to turn export off before EVERY patient, because exporting the data is destructive, and if we don't turn it off it here, exporting will automatically get run in the `generator.generatePerson(i)` call. So when you go to run it again a few lines down with `FhirDstu2.convertToFHIR(...)`, the results will be different the second time around.

Reply:
I'll take care of that though.